package/boot/uboot-mediatek/Makefile

```diff
@@ -303,7 +303,7 @@ define U-Boot/mt7981_glinet_gl-xe3000
 endef
 
 define U-Boot/mt7981_h3c_magic-nx30-pro
-  NAME:=H3C Magic NX30 Pro
+  NAME:=H3C Magic NX30 Pro (Stock U-Boot layout)
```

Comment:
Do we really need to add a new revision? Why not just merge those unused partitions? The official firmware seems to be able to restore those config partitions(need to be verified).

Reply:
The config and mac address will lose if the partitions are merged with the current scheme.